### ISSUE TITLE
Serenade doesn't load scripts recursively anymore

### ISSUE THREAD

**Issue by s-h-a-d-o-w** [serenadeai/custom-commands#7]
I just cloned this and realized that the commands in `all.js` only work if I move them to the root of `scripts`.

Side note: `commandOrControl` is apparently outdated.

```
os: 
  platform: win32
  release: 10.0.19043
version: 1.6.2
application: serenade
```

**Comment by tmacwill** [comment]
thanks for the report! we can reproduce scripts not loading recursively anymore, so we'll fix in our next release. `commandOrControl` should work across platforms, though. which keyboard layout are you using?

**Comment by s-h-a-d-o-w** [comment]
Great, thanks for the quick confirmation.

As for `commandOrControl` - I'm using the US keyboard layout.

**Comment by s-h-a-d-o-w** [comment]
I can confirm that the lack of recursion has been fixed with the release of `1.7.1`.
